{"error":"error: column \"name\" of relation \"customers\" does not exist\n    at /Users/umutcan/Documents/push-notification-system/node_modules/pg-pool/index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async /Users/umutcan/Documents/push-notification-system/src/routes/customer.js:100:20","level":"error","message":"Error in customer creation","timestamp":"2025-02-23T21:36:34.768Z"}
